header-footer-grid/Core/Settings/Manager.php

```diff
@@ -141,6 +141,38 @@ public function load( $group = null, \WP_Customize_Manager $customize_manager =
 				)
 			);
 
+			if ( isset( $arguments['settings'] ) ) {
+				foreach ( array_keys( $arguments['settings'] ) as $setting ) {
+					if ( $setting === 'default' ) {
+						$customize_manager->add_setting(
+							$id,
+							array(
+								'default'           => $arguments['preview_default'],
```

Comment:
Maybe I'm missing something, but is there a reason we couldn't simply use the 'default' key for the main setting? 🐑

Reply:
I remember that at some point the main setting wouldn't appear if I didn't add it here too (probably the issue was somewhere else). Checked again after deleting this call of `add_setting` and seems to work just fine :rocket: